### PR TITLE
Schedule creation UI enhancement

### LINKED ISSUE #460: "No Confirmation Needed" or "Confirmation Needed"
I like to confirm meeting times, but some people may not need to. We should expose an option in the short-term (and in the long-term expose this in onboarding) to be able to turn off confirmation (or we could choose to make that the default and have a "Confirmation Needed" option that you turn on). So that folks can allow people to grab time on their calendars instantly with no confirmation required.

### PR DESCRIPTION
**Pull request by devmount**
## Description of the Change

This change implements most of the remaining tasks to implement [the current design](https://www.figma.com/design/rDjrWQsgejQUyACjjXRwVa/TBA_2024_Q3_BookingConfirmationToggle?node-id=1-6&node-type=canvas&t=WwFZ4pgr9bGR79Rr-0) for the schedule creation section:

- Input labels
- Spacing
- Snackish bar
- Light / dark mode

## Benefits

image

a

## Applicable Issues

Closes #460

### CODE REVIEW

File: frontend/src/components/ScheduleCreation.vue [hunk 1]
```diff
@@ -660,12 +651,12 @@ watch(
               @change="toggleZoomLinkCreation"
               class="size-5 rounded-md"
             />
```
Comment (MelissaAutumn) [inline frontend/src/components/ScheduleCreation.vue:653]: This can be a separate PR if you want, but we should normalize these checkboxes with the SyncCard's checkboxes. (And then pull those checkboxes into SyncCard 😅 )

**Review by MelissaAutumn — APPROVED** [review]
Looks good, thanks!

File: frontend/src/components/ScheduleCreation.vue (hunk 1 above)
Comment (devmount) [reply frontend/src/components/ScheduleCreation.vue]: Ah that is a great suggestion! I forgot we already had styled checkboxes! I did it right away.

**Comment by devmount** [comment]
@MelissaAutumn If you like you can test again, I added the checkbox component now.

**Comment by MelissaAutumn** [comment]
Looks like I can still edit the weekday selections and the slug input while the schedule is disabled. I'll make a ticket so it doesn't block the merge of this though.

**Comment by devmount** [comment]
> Looks like I can still edit the weekday selections and the slug input while the schedule is disabled. I'll make a ticket so it doesn't block the merge of this though.

Good catch! Yes please create a ticket, I'll look into that
